app/javascript/application.js

```diff
@@ -1,4 +1,4 @@
 // Configure your import map in config/importmap.rb. Read more: https://github.com/rails/importmap-rails
-import "@hotwired/turbo-rails"
-import "controllers"
-import * as bootstrap from "bootstrap"
+import '@hotwired/turbo-rails'
+import 'controllers'
+import * as bootstrap from 'bootstrap' // eslint-disable-line no-unused-vars
```

Comment:
The changes to how we invoke `standard` in `package.json` found some violations we had been missing before.